pkg/utils/utils.go

```diff
@@ -75,8 +79,12 @@ func NewDynProgressBar(title string, quiet bool) (*mpb.Progress, *mpb.Bar) {
 
 // NewProgressCounter init a progress counter
 func NewProgressCounter(title string) (*mpb.Progress, *mpb.Bar) {
+	var quiet bool
```

Comment:
quiet := os.Getenv("PROGRESSBAR_DISPLAY") == "false"
or compare it directly in the `if` statment